test/Reddit.spec.ts

```diff
@@ -0,0 +1,120 @@
+            Pubkey: wallets[1].getPublicKeyString(),
+            Amount: 10,
+            TokenType: 1,
+            AccID: 3,
+            Path: "3",
+            nonce: 0,
+        };
+        testTokenInstance = await utils.registerToken(wallets[0]);
+        await testTokenInstance.transfer(Reddit.Address, 100);
+        await depositManagerInstance.depositFor(
+            Reddit.Address,
+            Reddit.Amount,
+            Reddit.TokenType,
+            Reddit.Pubkey
+        );
+        rollupCoreInstance = await RollupCore.deployed();
+        IMTInstance = await IMT.deployed();
+    })
+    it("Should Create Account for the User", async function () {
+        const createAccountInstance = await createAccount.deployed();
+        // Call to see what's the accountID
+        const accountId = await createAccountInstance.createPublickeys.call([User.Pubkey]);
+        assert.equal(accountId.toString(), "3");
+        // Actual execution
+        await createAccountInstance.createPublickeys([User.Pubkey]);
+
+        const tx = {
+            toIndex: 3,
+            tokenType: 1
+        } as CreateAccount;
+        const RollupUtilsInstance = await RollupUtils.deployed();
+        const signBytes = await RollupUtilsInstance.getCreateAccountSignBytes(tx.toIndex, tx.tokenType);
+        tx.signature = utils.sign(signBytes, wallets[0]);
+        const balanceRoot = await rollupCoreInstance.getLatestBalanceTreeRoot();
+        const accountRoot = await IMTInstance.getTreeRoot();
+        const ZeroAccount: Account = {
+            ID: 0,
+            tokenType: 0,
+            balance: 0,
+            nonce: 0,
+            burn: 0,
+            lastBurn: 0
+        }
+        const RedditAccountLeaf = await utils.createLeaf(Reddit)
+        const zeroHashes = await utils.defaultHashes(4);
+        const coordinator_leaves = await RollupUtilsInstance.GetGenesisLeaves();
+        const NewAccountSiblings: Array<string> = [
+            RedditAccountLeaf,
+            utils.getParentLeaf(coordinator_leaves[0], coordinator_leaves[1]),
+            zeroHashes[2],
+            zeroHashes[3],
+        ];
+        const NewAccountMP = {
+            accountIP: {
+                pathToAccount: User.Path,
+                account: ZeroAccount,
+            },
+            siblings: NewAccountSiblings,
+        };
+        const coordinatorPubkeyHash =
+            "0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563";
+
+        const userPDAsiblings = [
+            utils.PubKeyHash(Reddit.Pubkey),
+            utils.getParentLeaf(coordinatorPubkeyHash, coordinatorPubkeyHash),
+            zeroHashes[2],
+            zeroHashes[3],
+        ];
+        const userPDAProof = {
+            _pda: {
+                pathToPubkey: "3",
+                pubkey_leaf: { pubkey: User.Pubkey },
+            },
+            siblings: userPDAsiblings,
+        }
+        await createAccountInstance.processTx(
```

Comment:
Not sure why the gas reporter does not show the gas usage of this call.

Reply:
We can just call estimate_call on this and get the gas for now I think.

Reply:
Just realized since processTx is a view function. The estimate_call don't work. Got "createAccountInstance.processTx.estimate_call is not a function".